src/services/GoogleService.ts

```diff
@@ -0,0 +1,33 @@
+import CacheService from "./CacheService";
+import * as GoogleSpreadsheet  from "google-spreadsheet";
+
+
+class GoogleService {
+
+    private cache: CacheService;
+    private doc: GoogleSpreadsheet;
+    private sheet;
+    constructor() {
+        this.cache = new CacheService(1200);
+        this.doc = new GoogleSpreadsheet ("19WKHONdULFT3U8T-5dhguM5UgO71OzHAzwhZq1MKXaY");
```

Comment:
isn't it better to set this in the config file, rather than hardcoding the key it here?